python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -97,7 +98,40 @@ def test_simple_flow(lmdb_version_store_no_symbol_list, symbol):
     assert lmdb_version_store_no_symbol_list.list_symbols() == lmdb_version_store_no_symbol_list.list_versions() == []
 
 
-@pytest.mark.parametrize("special_char", ["$", ",", ":", "=", "@", "-", "_", ".", "~"])
+@pytest.mark.parametrize(
+    "special_char",
+    [
+        "$",
+        "@",
+        "=",
+        ";",
+        "/",
+        ":",
+        "+",
+        " ",
+        ",",
+        "?",
+        "\\",
+        "{",
+        "^",
+        "}",
+        "%",
+        "`",
+        "[",
+        "]",
+        '"',
+        "'",
+        "~",
+        "#",
+        "|",
+        "!",
+        "-",
+        "_",
+        ".",
+        "(",
+        ")",
+    ],
```

Comment:
Minor: making use of the fact a Python `str` is a `Sequence`, you can do `list("$@...")`, which is easier to review.